include/TMU.hpp

```diff
@@ -1,13 +1,167 @@
 #pragma once
 
-namespace BOARD_NAME {
+#include <Canopen/co_core.h>
+#include <EVT/io/CANopen.hpp>
+
+namespace TMU {
 
 /**
  * This is an example of a class for a board
  */
-class BOARD_NAME {
+class TMU {
 public:
+    static constexpr uint8_t NODE_ID = 0x09;
+
+    /**
+     * Gets the object dictionary
+     *
+     * @return an object dictionary
+     */
+    CO_OBJ_T* getObjectDictionary();
+
+    /**
+     * Gets the size of the Object Dictionary
+     *
+     * @return size of the Object Dictionary
+     */
+    uint16_t getObjectDictionarySize() const;
+
 private:
+    /**
+     * Stores the 4 16-bit temperature values.
+     */
+    uint16_t thermTemps[4] = {};
+
+    /**
+     * Object Dictionary Size
+     */
+    static constexpr uint16_t OBJECT_DICTIONARY_SIZE = 20;
+
+    /**
+     * CAN Open object dictionary
+     */
+    CO_OBJ_T objectDictionary[OBJECT_DICTIONARY_SIZE + 1] = {
+        // Sync ID, defaults to 0x80
+        {
+            CO_KEY(0x1005, 0, CO_UNSIGNED32 | CO_OBJ_D__R_),
+            nullptr,
+            (uintptr_t) 0x80,
+        },
+
+        /**
+         * Information about the hardware , hard coded sample values for now
+         * 1: Vendor ID
+         * 2: Product Code
+         * 3: Revision Number
+         * 4: Serial Number
+         */
+        {
+            .Key = CO_KEY(0x1018, 1, CO_UNSIGNED32 | CO_OBJ_D__R_),
+            .Type = nullptr,
+            .Data = (uintptr_t) 0x10,
+        },
+        {
+            .Key = CO_KEY(0x1018, 2, CO_UNSIGNED32 | CO_OBJ_D__R_),
+            .Type = nullptr,
+            .Data = (uintptr_t) 0x11,
+        },
+        {
+            .Key = CO_KEY(0x1018, 3, CO_UNSIGNED32 | CO_OBJ_D__R_),
+            .Type = nullptr,
+            .Data = (uintptr_t) 0x12,
+        },
+        {
+            .Key = CO_KEY(0x1018, 4, CO_UNSIGNED32 | CO_OBJ_D__R_),
+            .Type = nullptr,
+            .Data = (uintptr_t) 0x13,
+        },
+
+        /**
+         * SDO CAN message IDS.
+         * 1: Client -> Server ID, default is 0x600 + NODE_ID
+         * 2: Server -> Client ID, default is 0x580 + NODE_ID
+         */
+        {
+            .Key = CO_KEY(0x1200, 1, CO_UNSIGNED32 | CO_OBJ_D__R_),
+            .Type = nullptr,
+            .Data = (uintptr_t) 0x600 + NODE_ID,
+        },
+        {
+            .Key = CO_KEY(0x1200, 2, CO_UNSIGNED32 | CO_OBJ_D__R_),
+            .Type = nullptr,
+            .Data = (uintptr_t) 0x580 + NODE_ID,
+        },
+
+        /**
+         * TPDO 1 Settings
+         * 0. The number of sub indxes.
+         * 1. The COBID for the transmitting node
+         * 2. The transmission trigger 0xFE is asynchronous
+         * 3. The inhibit time
+         * 4. The event timer
+         */
+        {
+            .Key = CO_KEY(0x1800, 0, CO_UNSIGNED8 | CO_OBJ_D__R_),
+            .Type = nullptr,
+            .Data = (uintptr_t) 5},
```

Comment:
```suggestion
            .Data = (uintptr_t) 5,},
```
If you have a comma here in each element in the list, the autoformatter is much happier